Dashboard/pages/email_alerts.py

```diff
@@ -34,7 +34,7 @@ def email_alerts_page(boto_ses_client: ses_client) -> None:
     else:
         col1, col2 = st.columns([1, 8])
         with col1:
-            email_sign_up = st.button("Sign me up!")
+            email_sign_up = st.button("Sign up!")
```

Comment:
How come? It guess `Sign me up!` looks less professional

Reply:
It was on two lines so that was the main reason but also I normally see on websites just saying "Sign up"